packages/maskbook/src/plugins/Trader/trader/balancer/useTradeComputed.ts

```diff
@@ -19,8 +19,8 @@ export function useTradeComputed(
         const [swaps, tradeAmount, spotPrice] = swaps_
         const isExactIn = strategy === TradeStrategy.ExactIn
         const priceImpact = isExactIn
-            ? new BigNumber(inputAmount).div(tradeAmount).times('1e18').div(spotPrice).minus(1)
-            : new BigNumber(tradeAmount).div(outputAmount).times('1e18').div(spotPrice).minus(1)
+            ? new BigNumber(inputAmount).dividedBy(tradeAmount).multipliedBy('1e18').dividedBy(spotPrice).minus(1)
+            : new BigNumber(tradeAmount).dividedBy(outputAmount).multipliedBy('1e18').dividedBy(spotPrice).minus(1)
```

Comment:
Woo, why change our web3 library affects our usage on BigNumber?

Reply:
Ethers have got its own lib for big numbers and cannot use different libs for the same function. I'm removing `BigNumber.js`.